src/99-appendices/08-coordinate-systems.md

```diff
@@ -127,21 +127,21 @@ Unless specified explicitly in the sidecar file in the
+| NIHPD                               | Pediatric templates generated from the NIHPD sample. Available for different age groups (4.5–18.5 y.o., 4.5–8.5 y.o., 7–11 y.o., 7.5–13.5 y.o., 10–14 y.o., 13–18.5 y.o. This template also comes in either -symmetric or -asymmetric flavor. [https://www.bic.mni.mcgill.ca/ServicesAtlases/NIHPD-obj1](https://www.bic.mni.mcgill.ca/ServicesAtlases/NIHPD-obj1)                                                                                                                                                       |
 | OASIS30AntsOASISAnts                | [https://figshare.com/articles/ANTs_ANTsR_Brain_Templates/915436](https://figshare.com/articles/ANTs_ANTsR_Brain_Templates/915436)                                                                                                                                                                                                                                                                                                                                                                                     |
 | OASIS30Atropos                      | [https://mindboggle.info/data.html](https://mindboggle.info/data.html)                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-| Talairach                           | Piecewise linear scaling of the brain is implemented as described in TT88. [http://www.talairach.org/](http://www.talairach.org/)                                                                                                                                                                                                                                                                                                                                                                                      |
+| Talairach                           | Piecewise linear scaling of the brain is implemented as described in TT88. [https://talairach.org/](https://talairach.org/)                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
I've reached out to someone at the hosting institution to fix their TLS configuration, but no reason to hold this up for that.

```suggestion
| Talairach                           | Piecewise linear scaling of the brain is implemented as described in TT88. [http://talairach.org/](http://talairach.org/)                                                                                                                                                                                                                                                                                                                                                                                        |
```

Reply:
Thanks for doing the extra mile!
I wonder if the linkchecker could display a warning if the HTTP status code is not 200 so that redirections can be flagged for possible changes.